seqr/views/apis/anvil_workspace_api.py

```diff
@@ -174,6 +181,27 @@ def create_project_from_workspace(request, namespace, name):
         logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user,
                      detail=sample_ids)
 
+    # trigger dags
```

Comment:
I think it would be most readable to have all of this logic inside a single helper function, something like `_trigger_data_loading`